Lib/test/test_annotationlib.py

```diff
@@ -809,6 +809,24 @@ def f(x: int):
             {"x": "int"},
         )
 
+    def test_del_annotations(self):
```

Comment:
nit: this test doesn't use annotationlib, so it fits better in `test_type_annotations.py`.